docs/reference/slate/leaf.md

```diff
@@ -0,0 +1,99 @@
+# `Leaf`
+
+```js
+import { Leaf } from 'slate'
+```
+
+A leaf node in a Slate [`Document`](./document.md). Leaf nodes are always direct children of a [`Text`](./text.md) node and contains the textual content with associated [`Marks`](./mark.md).
```

Comment:
Can you remove the "nodes" qualifier here and throughout? We want to refer to leaves as simply "leaves" instead of as "leaf nodes", since they don't have any of the node methods.